### DISCUSSION TITLE
Cannot access K3s cluster after change in local IP (macOS)

### DISCUSSION THREAD

**Discussion by deors** [rancher-sandbox/rancher-desktop#5237] [General]
Hello,

I have a working K3s cluster managed by Rancher Desktop. It was working fine and I had several tools and apps in my sandbox environment.
As part of my installation, I have a Jenkins server to run builds using Kubernetes plug-in to schedule build jobs as pods. That worked also fine just using the local laptop IP to reach out Kubernetes API.

A couple of days ago, I tried to do some tests in a different location, hence with a different laptop IP, and I found out that a couple of things didn't update to reflect that environment change: the cluster advertising IP (as per Traefik ingress configuration, see below), and the certificate (also, see below). As a result, I cannot run builds anymore as I cannot reach out the API with the new IP.
I tried to reset the cluster, but it didn't fix anything (still the old IP and old certificate).

Is it possible to trigger a refresh on both the advertising IP for the cluster and refresh on the certificate?

Thanks in advance,
Jorge.
----
New IP: 192.168.1.47
Output from kubectl get ingress (the relevant piece):

NAME                                   CLASS     HOSTS   ADDRESS         PORTS   AGE
ingress.networking.k8s.io/ci-jenkins   traefik   *       192.168.1.117   80      3d20h

Output from kubectl command when using the laptop IP to reach out the API:

Unable to connect to the server: tls: failed to verify certificate: x509: certificate is valid for 10.43.0.1, 127.0.0.1, 192.168.1.117, 192.168.5.15, ::1, not 192.168.1.47

**Comment by jandubois** [comment]
I've transferred this question into an issue, so we can track it on the project board: #5263

I'm locking this discussion; all further feedback should happen on the issue.
